restful_api/myapi/api/schemas/course.py

```diff
@@ -2,10 +2,12 @@
 from myapi.extensions import ma, db
 
 from .prereq import PrereqSchema
+from .course_trainer import CourseTrainerSchema
 
 class CourseSchema(ma.SQLAlchemyAutoSchema):
 
     prereqs = ma.List(ma.Nested(PrereqSchema), required=False)
+    course_trainers = ma.List(ma.Nested(CourseTrainerSchema), required=False)
```

Comment:
https://marshmallow.readthedocs.io/en/stable/nesting.html @joelytj , for visibility on what this line is doing